pkg/kubelet/eviction/helpers.go

```diff
@@ -41,10 +41,16 @@ const (
 	message = "The node was low on compute resources."
 	// disk, in bytes.  internal to this module, used to account for local disk usage.
 	resourceDisk api.ResourceName = "disk"
+	// inodes, in bytes. internal to this module, used to account for local disk inode consumption.
```

Comment:
Good point, I got logically tripped up when using `df -ih`.  In practice, it is actually quite useful to express these in powers of 1024 ;-)

```
$ curl <stats>
     "rootfs": {
      "availableBytes": 15132864512,
      "capacityBytes": 52710469632,
      "inodesFree": 2592837,
      "inodes": 3276800
     },
$ df -i
Filesystem                Inodes  IUsed    IFree IUse% Mounted on
/dev/mapper/fedora-root  3276800 683963  2592837   21% /
$ df -ih
Filesystem              Inodes IUsed IFree IUse% Mounted on
/dev/mapper/fedora-root   3.2M  668K  2.5M   21% /
```